libs/common/src/constants/status.ts

```diff
@@ -0,0 +1,3 @@
+export const STATUS = {
+  INVALID: 'INVALID_STATUS',
+};
```

Comment:
If we are keeping this as a generic name then please can we go with the generic naming:
```suggestion
export const ERROR_KEY = {
  INVALID_STATUS: 'INVALID_STATUS',
};
```
`INVALID` isn't actually a status in this context so I don't think STATUS.INVALID is particularly clear

Reply:
this has been changed to INVALID_STATUS